### PR TITLE
Added 'tomorrow' property for Date class. (Addition to PR #578)

### PR DESCRIPTION
**Pull request by AlexeiGitH**
## Checklist

- [x] I checked the [**Contributing Guidelines**](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md) before creating this request.
- [x] New extensions are written in Swift 4.
- [x] New extensions support iOS 8.0+ / tvOS 9.0+ / macOS 10.10+ / watchOS 2.0+.
- [x] I have added tests for new extensions, and they passed.
- [x] All extensions have a **clear** comments explaining their functionality, all parameters and return type in English.
- [x] All extensions are declared as **public**.
- [x] I have added a [changelog](https://github.com/SwifterSwift/SwifterSwift/blob/master/CHANGELOG_GUIDELINES.md) entry describing my changes.

### CODE REVIEW

File: Tests/FoundationTests/DateExtensionsTests.swift [hunk 1]
```diff
@@ -957,6 +957,11 @@ final class DateExtensionsTests: XCTestCase {
         let date = Date()
         XCTAssertEqual(date.yesterday.timeIntervalSince(date), -86400.0)
     }
+
+    func testTomorrow() {
+        let date = Date()
+        XCTAssertTrue(date.tomorrow.isInTomorrow)
```
Comment (guykogus) [inline Tests/FoundationTests/DateExtensionsTests.swift]: For similar reasons to the ones discussed in https://github.com/SwifterSwift/SwifterSwift/pull/578 please use the same type of test as for `testYesterday`.

**Comment by LucianoPAlmeida** [comment]
Hey @AlexeiGitH :))
The Swiftlint is complaining about a trailing whitespace. Can you just make this little fix?

**Comment by AlexeiGitH** [comment]
> The Swiftlint is complaining about a trailing whitespace. Can you just make this little fix?
Where exactly that trailing space is?

**Comment by LucianoPAlmeida** [comment]
The SwifterSwiftBot message is showing Swiftlint issues on line 477 on DateExtensions.swift :))
> 4 Messages
> 📖	Thank you for submitting a pull request to SwifterSwift. The team will review your submission as soon as possible.
> 📖	iOS: Executed 533 tests, with 0 failures (0 unexpected) in 10.076 (13.197) seconds
> 📖	tvOS: Executed 502 tests, with 0 failures (0 unexpected) in 4.212 (7.444) seconds
> 📖	macOS: Executed 367 tests, with 0 failures (0 unexpected) in 1.517 (2.096) seconds
> ### SwiftLint found issues
> #### Warnings
> File	Line	Reason
> DateExtensions.swift	477	Lines should not have trailing whitespace.
> Generated by 🚫 [Danger](https://danger.systems/)

**Comment by guykogus** [comment]
@AlexeiGitH it looks like this got stuck, probably because it was triggered while GitHub was having serious issues yesterday. You might want to push an empty commit to try and trigger it again.
```
git commit --amend
git push -f
```

**Comment by guykogus** [comment]
@AlexeiGitH please resolve the conflicts, I made a small change to the CHANGELOG to remove `/files` at the end of my PR link. Sorry :)

**Pull request merged**